test/framework/options.py

```diff
@@ -366,6 +366,16 @@ def test_skip(self):
 
         self.assertEqual(len(glob.glob(toy_mod_glob)), 1)
 
+        for toy_mod in glob.glob(toy_mod_glob):
+            remove_file(toy_mod)
+
+        # check use of module_only parameter
+        test_ec_txt += "\nmodule_only = True\n"
+        write_file(test_ec, test_ec_txt)
+        self.eb_main(args, do_build=True, raise_error=True)
```

Comment:
@smoors You should tweak `args` here too, so it doesn't include `--skip` anymore (or `--force`, I guess), and also check that no software was installed?

Reply:
done in https://github.com/easybuilders/easybuild-framework/pull/4537/commits/430189c2dfd39b5ea494b66b06ab6acfa347851d